modules/network-observability-topology.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: PROCEDURE
+[id="network-observability-topology_{context}"]
+= Using Topology
+Topology provides a graphical representation of the network flows and the amount of traffic. As an administrator, you can monitor the traffic data across the application by using the topology view.
+
+== Working with Topology
+As an administrator, you can navigate to topology view by executing the following:
+
+. On the left navigation panel, click *Observe* → *Network Traffic*.
+. In the *Network Traffic* page, click the *Topology* tab.
+
+You can click each component in the topology to view the details and metrics of the component.
+
+== Configuring advanced options
```

Comment:
According to our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-procedures): In most cases, create your procedures as individual modules and include them in appropriate assemblies.

[Mod docs guidelines](https://redhat-documentation.github.io/modular-docs/#assembly-guidelines) states more specifically that subheadings can be used in concept or reference modules, not procedure modules. 

It looks like you have one procedure (working with topology) and one reference module (configuring advanced options). I would take your configuring advanced options content and make a reference module, and change the title - because you're not describing any steps to configure and it's just a list of options.

Reply:
Thanks. I will split them into separate modules.